sdk/servicebus/service-bus/samples/javascript/useProxy.js

```diff
@@ -28,17 +28,17 @@ async function main() {
   // Create an instance of the `HttpsProxyAgent` class with the proxy server information
   const proxyAgent = new HttpsProxyAgent(proxyInfo);
 
-  const sbClient = ServiceBusClient.createFromConnectionString(connectionString, {
-    webSocket: WebSocket,
-    webSocketConstructorOptions: {
-      agent: proxyAgent
+  const sbClient = new ServiceBusClient(connectionString, {
+    webSocketOptions: {
+      webSocket: WebSocket,
+      webSocketConstructorOptions: { agent: proxyAgent }
```

Comment:
Has this been tested?

Reply:
I didn't - I just moved the options to where they needed to go. I was able to test all the other samples but I was going to check in with @chradek to see how he normally does these.

Reply:
I'll come up with a plan for this. This is just moving the option to where it's now hosted.

Reply:
Issue created here: https://github.com/Azure/azure-sdk-for-js/issues/8031

Reply:
@richardpark-msft Please reach out to @ShivangiReja who had done the tests for proxy before